src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -83,6 +83,27 @@ public void storeMessage(Message message) {
 
     return messages;
   }
+  public List<Message> getAllMessages(){
+    List<Message> messages = new ArrayList<>();
+
+    Query query = new Query("Message")
+      .addSort("timestamp", SortDirection.DESCENDING);
+    PreparedQuery results = datastore.prepare(query);
+
+    for (Entity entity : results.asIterable()) {
+      try {
```

Comment:
The `try` seems to be missing a `catch`. You should be able to catch this error when running `mvn appengine:devserver`. It's important that the code is running successfully all the time.